pjrt/convert/utils/xplane_schema.zig

```diff
@@ -0,0 +1,554 @@
+const std = @import("std");
+const tf_op_utils = @import("tf_op_utils.zig");
+
+pub const kXlaAsyncOpLineName = "Async XLA Ops";
+
+pub const kHostThreadsPlaneName = "/host:CPU";
+pub const kGpuPlanePrefix = "/device:GPU:";
+pub const kTpuPlanePrefix = "/device:TPU:";
+pub const kCustomPlanePrefix = "/device:CUSTOM:";
+
+pub const HostEventType = enum(u16) {
+    kUnknownHostEventType = 0,
```

Comment:
those kPascalCase should be replace by snake_case.

alternatively instead of using different name in the enum and in the json,
 just use the human readable name in the enum by using the `@"ExecutorState::Process"` syntax.